ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -266,7 +263,7 @@ KEXT_STATIC int HandleVnodeOperation(
     }
     
     isDeleteAction = ActionBitIsSet(action, KAUTH_VNODE_DELETE);
-    isDirectory = VDIR == vnodeType;
+    isDirectory = vnode_isdir(currentVnode);
```

Comment:
Note that `vnode_isdir()` is a [fairly trivial operation](https://github.com/apple/darwin-xnu/blob/a449c6a3b8014d9406c2ddbdc81795da24aa7443/bsd/vfs/kpi_vfs.c#L1665) so given the distance between the other `vnode_vtype()` call and the use of the vtype here, I don't think the effort to avoid calling the accessor function again is warranted.